components/TopNavButtons.vue

```diff
@@ -7,6 +7,9 @@
             DISCOVER
           </v-btn>
         </v-col>
+        <v-col v-if="auth.address" cols="1" class="mt-2">
```

Comment:
let's use `auth.isLoggedIn` here as in the future it's possible the logged in status is more complex than the presence of an address